### PR TITLE
Consolidate OSS and Enterprise FAQs

### LINKED ISSUE #2600: InfluxDB Enterprise 1.9 -- 2 FAQ sections
Review and collapse... or perhaps dissect these and distribute into new pages within the Troubleshooting section?

Screen Shot 2021-05-25 at 10 53 31 PM

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2600

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
 ---
-title: InfluxDB frequently asked questions
-description: Common issues with InfluxDB OSS.
+title: InfluxDB Enterprise frequently asked questions
+description: Common issues with InfluxDB Enterprise
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:3]:
```suggestion
description: Common issues with InfluxDB Enterprise.
```

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 2]
```diff
@@ -79,6 +82,18 @@ Where applicable, it links to outstanding issues on GitHub.
 * [Does the precision of the timestamp matter?](#does-the-precision-of-the-timestamp-matter)
 * [What are the configuration recommendations and schema guidelines for writing sparse, historical data?](#what-are-the-configuration-recommendations-and-schema-guidelines-for-writing-sparse-historical-data)
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:83]: @pierwill, fyi, pushed up a link fix for "Why aren't data dropped after I've altered..."

Comment (kelseiv) [reply content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:83]: @pierwill, looks like there's a handful more link fixes we need to remove hyphens for...let's do a global search and fix, and then re-run a link audit at last stop before we push Enterprise 1.9 PR

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 3]
```diff
@@ -289,7 +304,8 @@ Acceptable Boolean syntax differs for data writes and data queries.
 
 For example, `SELECT * FROM "hamlet" WHERE "bool"=True` returns all points with `bool` set to `TRUE`, but `SELECT * FROM "hamlet" WHERE "bool"=T` returns nothing.
 
-{{% warn %}} [GitHub Issue #3939](https://github.com/influxdb/influxdb/issues/3939) {{% /warn %}}
+<!-- TODO: closed issue. Edit docs if necessary. -->
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:307]: nice! Are we tracking issues for potential edits or have a todo list for?

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 4]
```diff
@@ -779,7 +795,8 @@ time                  value	 precision_supplied  timestamp_supplied
 1970-01-01T02:00:00Z  6      h                   2
 ```
 
-{{% warn %}} [GitHub Issue #2977](https://github.com/influxdb/influxdb/issues/2977) {{% /warn %}}
+<!-- TODO: closed issue. Edit docs if necessary. -->
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:798]: same

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 5]
```diff
@@ -1251,3 +1268,73 @@ Increase the shard group duration for your data’s retention policy with the [`
 Second, temporarily lowering the [`cache-snapshot-write-cold-duration` configuration setting](/enterprise_influxdb/v1.9/administration/config/#cache-snapshot-write-cold-duration-10m).
 If you’re writing a lot of historical data, the default setting (`10m`) can cause the system to hold all of your data in cache for every shard.
 Temporarily lowering the `cache-snapshot-write-cold-duration` setting to `10s` while you write the historical data makes the process more efficient.
+## Where can I find InfluxDB Enterprise logs?
+
+On systemd operating systems service logs can be accessed using the `journalctl` command.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:1273]:
```suggestion
On systemd operating systems, service logs can be accessed using the `journalctl` command.
```

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 6]
```diff
@@ -1251,3 +1268,73 @@ Increase the shard group duration for your data’s retention policy with the [`
...
+This is the expected behavior if you haven't joined the meta node to the
+cluster.
+The `503` errors should stop showing up in the logs once you
+[join the meta node to the cluster](/enterprise_influxdb/v1.9/install-and-deploy/installation/meta_node_installation/#step-3-join-the-meta-nodes-to-the-cluster).
+
+## Why am I seeing a `409` error in some of my data node logs?
+
+When you create a
+[Continuous Query (CQ)](/enterprise_influxdb/v1.9/concepts/glossary/#continuous-query-cq)
+on your cluster every data node will ask for the CQ lease.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:1294]:
```suggestion
on your cluster, every data node will ask for the CQ lease.
```

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 7]
```diff
@@ -1251,3 +1268,73 @@ Increase the shard group duration for your data’s retention policy with the [`
...
+```
+
+## Why am I seeing `hinted handoff queue not empty` errors in my data node logs?
+
+```
+[write] 2016/10/18 10:35:21 write failed for shard 2382 on node 4: hinted handoff queue not empty
+```
+
+This error is informational only and does not necessarily indicate a problem in the cluster. It indicates that the node handling the write request currently has data in its local [hinted handoff](/enterprise_influxdb/v1.9/concepts/clustering/#hinted-handoff) queue for the destination node. Coordinating nodes will not attempt direct writes to other nodes until the hinted handoff queue for the destination node has fully drained. New data is instead appended to the hinted handoff queue. This helps data arrive in chronological order for consistency of graphs and alerts and also prevents unnecessary failed connection attempts between the data nodes. Until the hinted handoff queue is empty this message will continue to display in the logs. Monitor the size of the hinted handoff queues with `ls -lRh /var/lib/influxdb/hh` to ensure that they are decreasing in size.
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:1315]:
```suggestion
This error is informational only and does not necessarily indicate a problem in the cluster. It indicates that the node handling the write request currently has data in its local [hinted handoff](/enterprise_influxdb/v1.9/concepts/clustering/#hinted-handoff) queue for the destination node. Coordinating nodes will not attempt direct writes to other nodes until the hinted handoff queue for the destination node has fully drained. New data is instead appended to the hinted handoff queue. This helps data arrive in chronological order for consistency of graphs and alerts and also prevents unnecessary failed connection attempts between the data nodes. Until the hinted handoff queue is empty, this message will continue to display in the logs. Monitor the size of the hinted handoff queues with `ls -lRh /var/lib/influxdb/hh` to ensure that they are decreasing in size.
```

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 8]
```diff
@@ -1251,3 +1268,73 @@ Increase the shard group duration for your data’s retention policy with the [`
...
+
+## Why am I seeing `error writing count stats ...: partial write` errors in my data node logs?
+
+```
+[stats] 2016/10/18 10:35:21 error writing count stats for FOO_grafana: partial write
+```
+
+The `_internal` database collects per-node and also cluster-wide information about the InfluxDB Enterprise cluster. The cluster metrics are replicated to other nodes using `consistency=all`. For a [write consistency](/enterprise_influxdb/v1.9/concepts/clustering/#write-consistency) of `all`, InfluxDB returns a write error (500) for the write attempt even if the points are successfully queued in hinted handoff. Thus, if there are points still in hinted handoff, the `_internal` writes will fail the consistency check and log the error, even though the data is in the durable hinted handoff queue and should eventually persist.
+
+
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:1327]:
```suggestion

```

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md [hunk 9]
```diff
@@ -1251,3 +1268,73 @@ Increase the shard group duration for your data’s retention policy with the [`
...
+The controlling configuration settings are in the `[hinted-handoff]` section of the file. `max-size` is the total size in bytes per hinted handoff queue. When `max-size` is exceeded, all new writes for that node are rejected until the queue drops below `max-size`. `max-age` is the maximum length of time a point will persist in the queue. Once this limit has been reached, points expire from the queue. The age is calculated from the write time of the point, not the timestamp of the point.
+
+## Why am I seeing `unable to determine if "hostname" is a meta node` when I try to add a meta node with `influxd-ctl join`?
+
+Meta nodes use the `/status` endpoint to determine the current state of another metanode. A healthy meta node that is ready to join the cluster will respond with a `200` HTTP response code and a JSON string with the following format (assuming the default ports):
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:1336]:
```suggestion
Meta nodes use the `/status` endpoint to determine the current state of another meta node. A healthy meta node that is ready to join the cluster will respond with a `200` HTTP response code and a JSON string with the following format (assuming the default ports):
```

File: content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md (hunk 3 above)
Comment (pierwill) [reply content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md:310]: https://github.com/influxdata/docs-v2/issues/2630

**Pull request merged**